lib/sisimai/bite/email.rb

```diff
@@ -0,0 +1,53 @@
+module Sisimai
+  module Bite
+    module Email
+      # Sisimai::Bite::Email- Base class for Sisimai::Bite::Email::*
+      class << self
+        # Imported from p5-Sisimail/lib/Sisimai/Bite/Email.pm
+        require 'sisimai/bite'
+        require 'sisimai/rfc5322'
+
+        # @abstract Flags for position variable
+        # @return   [Hash] Position flag data
+        # @private
+        def INDICATORS
+          return {
+            :'deliverystatus' => (1 << 1),
+            :'message-rfc822' => (1 << 2),
+          }
+        end
+        def headerlist; return []; end
+        def pattern;    return []; end
+
+        # @abstract MTA list
+        # @return   [Array] MTA list with order
+        def index
+          return %w|
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->